projects/quml-library/src/lib/services/viewer-service/viewer-service.ts

```diff
@@ -58,7 +58,7 @@ export class ViewerService {
     this.isSectionsAvailable = parentConfig?.isSectionsAvailable;
     this.src = config.metadata.artifactUrl || '';
     this.questionSetId = config.metadata.identifier;
-    this.questionSetEvaluable = this.serverValidationCheck(config.metadata?.eval);
+    this.questionSetEvaluable = this.serverValidationCheck(config.metadata?.evalMode);
```

Comment:
@pavankumar0408  Can you set the default value of `questionSetEvaluable` to false instead of type `any` and in `serverValidationCheck` method re-assign the value to true based on condition check.

Reply:
done